profiles/package.mask

```diff
@@ -33,6 +33,17 @@
 
 #--- END OF EXAMPLES ---
 
+# Andreas Sturmlechner <asturm@gentoo.org> (2025-04-01)
+# Package destabilisation mask: No KF6-based- for ffmpeg-7 compatible
```

Comment:
I like this and I'm going to suggest it in future. It solves qualms I have with doing it. Thanks!